executor/src/main/java/com/redhat/service/bridge/executor/GlobalConfigProvider.java

```diff
@@ -0,0 +1,24 @@
+package com.redhat.service.bridge.executor;
+
+import java.util.Optional;
+
+import javax.enterprise.context.Dependent;
+import javax.ws.rs.Produces;
+
+import org.eclipse.microprofile.config.inject.ConfigProperty;
+
+import com.redhat.service.bridge.actions.GlobalConfig;
+
+@Dependent
```

Comment:
Why dependent? Isn't this a single instance for the whole application?

Reply:
I'm following the reference implementation from quarkus guide https://quarkus.io/guides/cdi-reference#default_beans